.github/workflows/ubi9-build.yaml

```diff
@@ -4,6 +4,7 @@ name: Build of UBI 9 based Developer Images
 on:
   push:
     branches: [ main ]
+  workflow_dispatch:
```

Comment:
Maybe I just don't understand, but is it correct to have `workflow_dispatch:` with no value?

Reply:
I don’t think we need to define any inputs for triggering a new build manually, so the workflow_dispatch event can remain without any parameters.